docs/architecture/modernize-desktop/example-migration-to-net-core.md

```diff
@@ -0,0 +1,365 @@
+
+![API contract](media/example-migration-to-net-core/api-contract.png)
+
+We click on *Finish* and after a while, we will see the generated code.
+
+We can see three generated documents:
+
+1. Getting Started: just a link to GitHub to provide some info on WCF
+2. `ConnectedService.json`: configuration parameters to connect to the service.
+3. `Reference.cs`: this is the actual WCF client code.
+
+![Generated documents](media/example-migration-to-net-core/generated-documents.png)
+
+If we compile again, we see many errors coming from `.cs` files inside the Helper
+folder. This folder was present in the .NET Framework version but not included
+in the old .csproj. But with the new SDK project style every code file present
+underneath the project file location is included by default, meaning that the
+new .NET Core project tries to compile the files inside the Helper folder. Since
+it is not needed, we can safely delete it.
+
+We compile again but when we execute the application, we see no images of
+products. The problem is that now the path to the files has changed slightly and
+we need to add another level of depth in the path, updating from:
+
+```C#
+string image_name = Environment.CurrentDirectory + "\\..\\..\\Assets\\Images\\Catalog\\" + catalogItems.Picturefilename;
+```
+
+to
+
+```C#
+string image_name = Environment.CurrentDirectory + "\\..\\..\\..\\Assets\\Images\\Catalog\\" + catalogItems.Picturefilename;
+```
+
+After this change, we check that the application launches and performs as
+expected on .NET Core.
+
+## Migrating a WPF Application
+
+We will use the Shop.ClassicWPF sample application to perform the migration.
+Here is a screenshot of the app before migration.
+
+![Sample app before migration](media/example-migration-to-net-core/before-migration.png)
+
+This application uses a local SQLExpress database to hold the product catalog
+information. This database is accessed directly from the WPF application.
+
+We must first update the .csproj file to the new SDK style used by the .NET Core
+applications. We'll follow the same steps described in the Windows Forms
+migration, we unload the project, open the .csproj file and update its contents
+and finally reload the project.
+
+In this case, we delete all the content of the .csproj file and replace it with:
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
+    <PropertyGroup>
+        <OutputType>WinExe</OutputType>
+        <TargetFramework>netcoreapp3.0</TargetFramework>
+        <UseWPF>true</UseWPF>
+        <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    </PropertyGroup>
+</Project>
+```
+
+If we reload the project and compile, we get the following error:
+
+![Compile error](media/example-migration-to-net-core/compile-error.png)
+
+Since we have deleted all the .csproj contents we have lost a project reference
+specification present in the old project. We just need to add this line to
+.csproj to include it:
```

Comment:
```suggestion
Since you've deleted all the *.csproj* contents, you've lost a project reference
specification present in the old project. You just need to add this line to the
*.csproj* file to include the project reference back:
```